### ISSUE TITLE
[mesheryctl] `model search` command not working for some model names.

### ISSUE THREAD

**Issue by Jougan-0** [meshery/meshery#11319]
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

### Current Behavior

When we run `mesheryctl model search [model-name]` it logs no model found even though model exist with this name.
### Screenshots/Logs

```
./mesheryctl model search aws-dynamodb-controller
No model(s) found
```
### Environment

- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- 📺 [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- ⌨️ [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- ⌨️ [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

**Comment by EleisonC** [comment]
I would like to take this one if available @Jougan-0

**Comment by Jougan-0** [comment]
> I would like to take this one if available @Jougan-0

Good, a final check before assigning this to you. Did you setup mesheryctl locally?

**Comment by EleisonC** [comment]
Yes, I have setup mesheryctl locally. @Jougan-0

**Comment by Jougan-0** [comment]
Here you go @EleisonC .

**Comment by abhisheksatpathy** [comment]
hi @Jougan-0 , can i work on this please?

**Comment by yash37158** [comment]
I don’t believe there are any updates on the issue. I am opening this for new contributors

**Comment by bomanaps** [comment]
Hello, Can I pick this.
I have Meshery running

**Comment by yash37158** [comment]
Here you go @bomanaps

**Comment by lekaf974** [comment]
@bomanaps still working on it ?

**Comment by bomanaps** [comment]
No
